src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2357,4 +2353,12 @@ Uninstall the following packages: {0}?</value>
   <data name="PackagePathViewModel_Standard_Library" xml:space="preserve">
     <value>Standard Library</value>
   </data>
-</root>
+  <data name="PackagePathViewSummary2" xml:space="preserve">
+    <value>The default save location is the first path that is not the Standard Library location</value>
```

Comment:
what do you think about default `install` location instead of `save`?

Reply:
Sounds good to me.